modules/test/playwright/tests/osb-faro-web/page.spec.ts

```diff
@@ -246,26 +239,8 @@ test(
 	},
 
 	async ({apiHelpers, assetPublisherPage, page, pageEditorPage}) => {
-		const channelName = 'My Property ' + getRandomString();
-
-		const siteName = 'My Site ' + getRandomString();
-
-		const blogName = 'My Blog ' + getRandomString();
-
-		const pageName = 'My Blog Page ' + getRandomString();
-
-		const site = await apiHelpers.headlessSite.createSite({
-			name: siteName,
-		});
-
-		await test.step('Connect the DXP to AC', async () => {
-			await syncAnalyticsCloud({
-				apiHelpers,
-				channelName,
-				page,
-				siteName,
-			});
-		});
+		const blogName = 'My Blog ' + randomString;
+		const blogPage = 'My Blog Page ' + randomString;
```

Comment:
@rafaellajordao, does that make sense we rename it as `blogPageTitle`?

Reply:
yeahh I'll fix it!